lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -942,7 +955,8 @@ enum {
 // helper function to get parent scope of debug instruction, to be used
 // to determine with which compile unit the particular instruction relates
 inline bool hasDbgInstParentScopeIdx(const uint32_t Kind,
```

Comment:
This change can be avoided if we change the operand order to be line,scope,column. Is there a reason why we need to have column before scope?

Reply:
I understand this, but decided that we should not align specification for some implementation simplifications.
The order `line, column, scope` is common/"unified" throughout the specification, so I think we should put the arguments in alike way.